app/views/departments/index.html.erb

```diff
@@ -0,0 +1,80 @@
+<!--<h1>Listing Tickets</h1>-->
+
+<!--<table>-->
+<!--  <tr>-->
+<!--    <th>Subject</th>-->
```

Comment:
Remove commencted code. No commented code should be present

Reply:
Done